typescriptSrc/world.ts

```diff
@@ -72,166 +72,177 @@ module world {
         }
     }
 
+    function isBool(val: Value) : boolean {
```

Comment:
This isn't write.  Add the line if( ! val.isString() ) return false; as the first line.